cmd/namespace.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"encoding/json"
+
+	log "github.com/Sirupsen/logrus"
+	"k8s.io/kubernetes/pkg/api"
+)
+
+type namespacehandler struct {
+	storage *store
+}
+
+// OnAdd called with a namespace is added to k8s
+func (h *namespacehandler) OnAdd(obj interface{}) {
+	ns, ok := obj.(*api.Namespace)
```

Comment:
I'd keep the cast check here for consistency with other places. We can change that later everywhere if we want to. Same as kubernetes code base btw.